magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/poll.ftl

```diff
@@ -0,0 +1,56 @@
+[#assign pollSelector = state.getSelector()!]
+
+[#if pollSelector?has_content]
+  [#assign targetPoll = cmsfn.contentByPath("/"+pollSelector, "polls")!]
+
+  [#if targetPoll?has_content]
+    [#assign image = damfn.getAsset(targetPoll.imageLink)!]
+
+    <#--  
+      The image of a poll can probably be deleted in DAM 
+      despite the fact that the poll is still existing.  
+    -->
+    [#if image?has_content && image.link?has_content]
+      <img src="${image.link}">
+    [#else]
+      <img src="${ctx.contextPath}/.resources/voting-contentApp/resources/image/default.jpg">
+    [/#if]
+
+    [#if targetPoll.name?has_content]
+      <h1>${targetPoll.name}</h1>
+    
+      <p>${targetPoll.description}</p>
+
+      Poll will be expired: ${targetPoll.expiryDate?string["HH:mm d MMMM yyyy"]}<br><br>
+
+      <form action="" method="post">
+        <input type="text" name="Full Name" placeholder="Full Name (*)" required><br><br>
+        <input type="email" name="Email" placeholder="Email"><br><br>
+        
+        ${targetPoll.question}<br><br>
```

Comment:
In the future, we may need to do some update if we intend to have multiple question. It's good to think about it, since changing and controlling UI logic is cumbersome, there's one solution that we can wrap it into model and simplify UI logic check.

Reply:
Thanks for your advice.